sites/mosul/configs/openmrs/initializer_config/liquibase/liquibase.xml

```diff
@@ -181,6 +233,17 @@
 
     <!-- RENAL FUNCTION: Delete Urine routine and microscopy -->
     <changeSet id="delete-concept-set-renal-function" author="vasharma05">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="true">
+                SELECT CASE WHEN COUNT(*) > 0 THEN 'true' ELSE 'false' END
```

Comment:
Same applies here. The other `sqlCheck` uses 1/0

Reply:
Done